### ISSUE TITLE
Make apatite more common [E6:E]

### ISSUE THREAD

**Issue by CKNoah** [EnigmaticaModpacks/Enigmatica6#4437]
### Describe your suggestion

Increase the vein size of apatite, and the chance of spawning per chunk.

### Why would you like this added/changed?

I'm currently stuck on the quest for invar dust, due to the fact that I need apatite for phyto-grow. After looking for hours, I still haven't found any. I even mined an entire chunk until reaching y64. I searched in creative, to only rarely find it, and when I did, it was quite small. Maybe I'm doing something wrong, I don't know. If I am, please let me know.

**Comment by MuteTiefling** [comment]
You'll want to look in mountains. It spawns higher in the world, so it's largely at the surface/mountains. A quick wander around a nearby mountain got me enough to make several stacks of phytogro early on.

**Comment by CKNoah** [comment]
Thanks. Could you maybe add that to a tooltip or something? I get this is an expert pack, but there are still some stuff not everyone is experienced with.

**Comment by theboo** [comment]
for any emendatus enigmatica ore, click on the ore in jei and look at the info tab, it will show you a y-level distribution for the ore in each dim... the center of that y-level will have the highest yield with it getting more rare at the extremes

**Comment by theboo** [comment]
i think maybe you're just having bad luck... 3 veins of it right on this mountain side first mountain i flew too, and this ore is very high yield apatite per ore, mining 3 blocks with no fortune have me 16 apatite
![image](https://user-images.githubusercontent.com/3585761/159104107-8ff59a36-4eb7-4673-98d4-1e937cd6429f.png)

**Comment by HaxtonSale1** [comment]
the jei info does not appear on chunks; maybe that info could be applied to chunks

**Comment by MuteTiefling** [comment]
@Ridanisaurus what do you think?

**Comment by CKNoah** [comment]
> for any emendatus enigmatica ore, click on the ore in jei and look at the info tab, it will show you a y-level distribution for the ore in each dim... the center of that y-level will have the highest yield with it getting more rare at the extremes
> 
> ![image](https://user-images.githubusercontent.com/3585761/159103956-0408fdc0-787c-4f87-afa0-d997d8349e0e.png)
Yes, by asking for a tooltip, I meant the 'apatite spawns in mountains tip'
 to go in there

**Comment by MuteTiefling** [comment]
It's not just 'in mountains' though, that's what that information is telling you. It's anywhere above y64. Which could just be a hill.

**Comment by CKNoah** [comment]
I said "tooltip" as in "tip", or a suggestion.
